tests/test_statusline.py

```
@@ -0,0 +1,1953 @@
+
+    def test_locale_currency_not_supported_falls_back_to_usd(self) -> None:
+        """ロケール通貨がローカル定義にないならUSD"""
+        with patch("statusline._get_currency_from_locale", return_value="XYZ"):
+            assert statusline._resolve_currency(None) == "USD"
+
+
+class TestExtractVersion:
+    """_extract_version のテスト"""
+
+    @pytest.mark.parametrize(
+        "model_id, expected",
```

Ruff (0.15.4)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)